journal/modernized/31-chapter-28.adoc

```diff
@@ -589,7 +589,7 @@ but knows the most inward thoughts and imaginations of every heart.
 I was carried forward to unusual length,
 and without any prospect of drawing to a conclusion;
 the blessed truths of the gospel unfolding in continued
-succession to the view of my mind for about an hour and a-half:
+succession to the view of my mind for about an hour and a half:
```

Comment:
I started editing in the original when I meant to be in the modernized. Does the dash need to be eliminated in the original?